### PR TITLE
Part 5: Pipelines - Blueprints to fuel consistency and enablement PR

### PR DESCRIPTION
**Pull request by wpschaub**
Part 5 of the pipeline series, which intrioduces app-type templates and closes the chapter on continuous integration pipelines (CI). It introduces the theme of **blueprints** and associated **templates** represented as re-usable LEGO blocks, ranging from a bag of "do it yourself" blocks, to complex and detailed kits, such as the Imperial Star Destroyer.

### CODE REVIEW

**Comment by wpschaub** [comment]
@said-akram-wcbbc can you review this latest article and proposed content improvements? I would like to ensure that we represent the application-type templates correctly. Any objections if I add a reference to you, by name, and highlight that you are one of our pipeline champions, responsible for the app-type templates.

@alexbunardzic I will loop you and other reviewers in as soon as Said and I are happy with the latest installment.

File: content/moving-hundreds-of-pipeline-snowflakes-part5.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+
+Like the building-blueprint counterpart, our **generic** and **application-type** pipeline blueprints define templates that allow rapid and consistent creation of unlimited number of continuous delivery pipeline copies. Let us have a look at a few of our blueprints.
+
+---
+
+# Generic blueprints
+
+We have defined two generic blueprints. One defines a one stage, multiple jobs pipeline, allowing parallel runs on separate agents, each with multiple steps. The simpler and more recommended blueprint defines a one stage, one job, with multiple steps pipeline. It keeps things simple, running everything on one agent. 
+
+Both blueprints call our **bootstrap** template, which in injects other templates introducing DevSecOps scans such as SonarQube and WhiteSource, as well as custom built **building code** scripts and products, based on queue-time parameters.
```
Comment (said-akram-wcbbc) [inline content/moving-hundreds-of-pipeline-snowflakes-part5.md]: which <s>**in**</s> injects other templates

Comment (said-akram-wcbbc) [reply content/moving-hundreds-of-pipeline-snowflakes-part5.md]: Hi @wpschaub,
Great job!
No objection to mention my name.

**Review by said-akram-wcbbc — DISMISSED** [review]
Very insightful article!
Approved

**Review by said-akram-wcbbc — APPROVED** [review]

**Review by solicitfeedback — APPROVED** [review]

**Pull request merged**